igcommit/git.py

```diff
@@ -147,11 +143,13 @@ def get_changed_files(self):
             ]).decode('utf-8')
             changed_files = []
             for line in output.splitlines():
-                line_split = line.split(None, 5)
-                assert len(line_split) == 6
-                assert line_split[0].startswith(':')
-                file_mode = line_split[1]
-                file_path = line_split[5]
+                assert line.startswith(':')
+                line_split = line.split('\t', 1)
+                assert len(line_split) == 2
+                metadata, file_path = line_split
+                metadata_split = metadata.split(' ')
+                assert len(metadata_split) == 5
+                file_mode = metadata_split[1]
```

Comment:
I don't think this will work. On my machine there are not tabs at least. See also https://github.com/innogames/igcommit/pull/19

See the following as an example where 09 is hex for backslash:

```
echo -e "a\tb" | xxd
00000000: 6109 620a                                a.b.
```

No see the example output of a diff:

```
git diff-tree -r --root --no-commit-id --break-rewrites --no-renames --diff-filter=AM b52c42462a70215fab0f41f4dd4daad5e68ac412 | xxd
00000000: 3a31 3030 3634 3420 3130 3036 3434 2062  :100644 100644 b
00000010: 6464 3833 3363 3461 6566 6436 3965 3761  dd833c4aefd69e7a
00000020: 3061 3135 3563 3833 3063 3863 3761 3439  0a155c830c8c7a49
00000030: 3864 3066 6338 6220 6135 6664 3437 3466  8d0fc8b a5fd474f
00000040: 6233 3435 3962 3764 3435 6163 3730 3135  b3459b7d45ac7015
00000050: 6462 6566 6566 3736 6634 6338 3666 6333  dbefef76f4c86fc3
00000060: 204d 0969 6763 6f6d 6d69 742f 6769 742e   M.igcommit/git.
00000070: 7079 0a                                  py.
```

Reply:
Okay, I'll remove this commit.

Reply:
Removed